src/test/histogram_perf.c

```diff
@@ -100,8 +100,12 @@ int main() {
       gettimeofday(&finish, NULL);
       double elapsed = finish.tv_sec - start.tv_sec;
       elapsed += (finish.tv_usec/1000000.0) - (start.tv_usec/1000000.0);
+      if(cnt != 0)
```

Comment:
the bodies of the `if` and `else` you added here should be indented to make it clear they are inside of the `if`/`else`

```
if(cnt != 0)
  printf(...);
else
  printf(...);
```